zstacklib/zstacklib/storage/iscsi/session.py

```
@@ -0,0 +1,472 @@
+        session_id: Session ID to rescan
+        timeout: Command timeout in seconds (default 30)
+        
+    Raises:
+        RescanError: If rescan fails
+        
+    Example:
+        >>> session = login('192.168.1.100', 'iqn.2020-01.com.example:storage')
+        >>> rescan_session(session.session_id)
+    """
+    cmd = "timeout {} iscsiadm -m session -r {} --rescan".format(timeout, session_id)
+    r, o, e = bash.bash_roe(cmd)
```

Ruff (0.15.2)

[warning] Unpacked variable `o` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)